packages/server/src/users/users.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+// export type User = {
```

Comment:
NIT: Could probably remove the commented out code, or added explanation on why the commented code should remain

Reply:
I think probably you need to pull the user.service.ts from the main branch.